src/Autonomous.cpp

```diff
@@ -183,6 +198,52 @@ void Autonomous::autonomyIter()
 			}
 		}
 
+		const points_t lidar_scan = readLidarScan();
+		bool should_replan = ((clock_counter++) % 20 == 0); // TODO make this configurable
+		if (should_replan) {
+			plan_base = pose;
+			plan_idx = 0;
+			point_t point_t_goal;
+			point_t_goal.topRows(2) = driveTarget.topRows(2);
+			point_t_goal(2) = 1.0;
+			point_t_goal = toTransform(pose) * point_t_goal;
+			double goal_radius = 2.0;
+			plan = getPlan(lidar_scan, point_t_goal, goal_radius);
+		}
+
+		int c;
+		while ((c = viz_window.pollWindowEvent()) != -1) {}
```

Comment:
Since `c` isn't used anywhere, I think this can be simplified to `while (viz_window.pollWindowEvent() != -1)`